examples/config-tinygo/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	spinconfig "github.com/fermyon/spin/sdk/go/config"
+	spinhttp "github.com/fermyon/spin/sdk/go/http"
+)
+
+func init() {
+	spinhttp.Handle(func(w http.ResponseWriter, r *http.Request) {
+		if val, err := spinconfig.Get("message"); err != nil {
```

Comment:
Would you mind adding a comment here describing where the value is coming from?
Thanks!